voa/copula.py

```diff
@@ -26,23 +28,25 @@ def create_Q_grid(n=10):
 
     return grid
 
+
 @numba.njit(parallel=True)
 def calculate_copula_part(t, c):
     n = len(t)
-    n_inv = 1 / n # n**-1 ## numba doesn't get n**-1 xD
+    n_inv = 1.0 / float(n)
     c = np.zeros(c.shape)
     for i in numba.prange(1, int(n / 2) + 2):
         for j in numba.prange(int(n / 2) + 2):
-            # c[i, j] = c[i - 1, j] + (j >= t[i - 1]) * n_inv
             c[i, j] = c[0, j] + n_inv * np.sum(j >= t[:i])
     return c
 
+
 @numba.njit
 def wn(n, c, i, j):
```

Comment:
I can maybe accept very short parameter names, but please name function verbosely.

Reply:
The same as above applies here. The name wn come from the paper. We can change all the names, but the question is if we should. @mp360288 wdyt?

Reply:
makes sense then if it comes from paper. For readability imo you can do two things:
- add docstring
- differentiate between public and private API - I guess this `wn` function will never be called by user?